components/IngredientsList.js

```diff
@@ -0,0 +1,142 @@
+import ingredients from "@/assets/ingredients.json";
+import styled from "styled-components";
+
+const IngredientsList = () => {
+  const flavorColors = {
+    Bitter: "#D2691E",
+    Creamy: "#FFFACD",
+    Fresh: "#98FB98",
+    Herbal: "#228B22",
+    Pungent: "#FF6347",
+    Salty: "#F0E68C",
+    Savory: "#D2B48C",
+    Spicy: "#cf2210",
+    Sour: "#d9df0c",
+    Sweet: "#FFB6C1",
+    Tangy: "#FFA07A",
+    Umami: "#188989",
+  };
+  return (
+    <>
+      <Title>Ingredients Overview</Title>
+      <Container>
+        <List>
+          {ingredients.map((ingredient) => (
+            <ListItem
+              key={ingredient._id}
+              $color={flavorColors[ingredient.flavor]}
+            >
+              <ImageWrapper>
+                <StyledImage imgUrl={ingredient.imgUrl} />
```

Comment:
Your should be using the [Image component of Next.js](https://nextjs.org/docs/app/api-reference/components/image) and style it to your liking. 

After having changed it, you need to add some props:

```suggestion
<StyledImage
                  src={ingredient.imgUrl}
                  fill
                  sizes="(max-width: 768px) 100vw, (max-width: 1200px) 50vw, 33vw"
                  alt={ingredient.name}
                />
```

`src` and `alt` are needed in any case.
`fill` and `sizes` work nicely, since you have a wrapping container with `position: relative`;